# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/grub2/openfirmware-no-colors.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

With color palette set some systems, like Sun U5/U10 are black on black, ...

--- grub-2.06/grub-core/kern/ieee1275/cmain.c.vanilla	2021-12-08 18:16:04.176739527 +0100
+++ grub-2.06/grub-core/kern/ieee1275/cmain.c	2021-12-08 18:16:32.273738406 +0100
@@ -60,6 +60,7 @@
 
 #ifdef __sparc__
   grub_ieee1275_set_flag (GRUB_IEEE1275_FLAG_NO_PARTITION_0);
+  grub_ieee1275_set_flag (GRUB_IEEE1275_FLAG_CANNOT_SET_COLORS);
 #endif
 
   grub_ieee1275_finddevice ("/", &root);
